server/server.js

```diff
@@ -23,24 +24,27 @@ exports = {
             raw: args.data.ticket.description_text
            }
         }
-      }).then( data => {
-        console.log('Successfully created the BitBucket issue for the freshdesk ticket');
-        saveMapping({
-          ticketID: args.data.ticket.id,
-          issueID: data.response.id,
-          issueUUID: data.response.reporter.uuid
-        }).then( function success(){
-          console.info('Successfully set the mapping in the db');
-        },error =>{
-          console.error('Error: Failed to set the mapping in the db');
-          console.error(JSON.stringify(error));
-        });
-      },error => {
-        console.error('Error: Failed to create the BitBucket issue for the freshdesk ticket');
-        console.error(JSON.stringify(error));
+      })
+
+       if(data.status !== 201 ) throw new Error("Error: Failed to create the BitBucket issue for the freshdesk ticket");
+       console.log('Successfully created the BitBucket issue for the freshdesk ticket');
+       const storeDataStatus =  await saveMapping({
+        ticketID: args.data.ticket.id,
+        issueID: data.response.id,
+        issueUUID: data.response.reporter.uuid
       });
+
+      if(storeDataStatus[0].Created === false && storeDataStatus[1].Created === false) throw new Error("Error: Failed to set the mapping in the db");
+      console.info('Successfully set the mapping in the db');
+
+
+    } catch(error){
+      console.error(`${error.message}`);
```

Comment:
Unnecessary string interpolation
```suggestion
      console.error(error.message);
```